File: DSA/Pragya_Agarwal/Day2_Question1.py
threshold = 3
arrival_times = [-1, -3, 4, 2]

on_time_count = 0

for time in arrival_times:
    if time <= 0:
        on_time_count += 1

if on_time_count < threshold:
    print("YES")
else:
    print("NO")

File: DSA/Aditi-Sharma/Day2/Day2_question1.java
import java.util.*

public class isClassCancelled{
    public static String isClassCancelled(int threshold_no, int[] arrival_times) {
       int k = 0;
       for (int i = 0; i < arrival_times.length; i++) {
           if (arrival_times[i] <= 0) {
               k++;
           }
       }
       if (k < threshold_no) {
           return "YES";
       } else {
           return "NO";
       }
   }
   
   public static void main(String[] args) {
       int threshold1 = 3;
       int[] arrivalTimes1 = {-1, -3, 4, 2};
       System.out.println(isClassCancelled(threshold1, arrivalTimes1)); // Output: YES
   }
